test/util/testnode/config.go

```diff
@@ -166,3 +172,19 @@ func DefaultTendermintConfig() *tmconfig.Config {
 
 	return tmCfg
 }
+
+func DefaultAppCreator() srvtypes.AppCreator {
+	return func(_ log.Logger, _ tmdb.DB, _ io.Writer, _ srvtypes.AppOptions) srvtypes.Application {
+		encodingConfig := encoding.MakeConfig(app.ModuleEncodingRegisters...)
+		return app.New(
+			log.NewNopLogger(),
+			tmdb.NewMemDB(),
+			nil,            // trace store
+			0,              // invCheckPerid
+			encodingConfig, //encoding config
+			0,              // upgrade height v2
+			simapp.EmptyAppOptions{},
+			baseapp.SetMinGasPrices(fmt.Sprintf("%v%v", appconsts.DefaultMinGasPrice, app.BondDenom)),
```

Comment:
this line is the fix

Reply:
when did this break? out of curiosity

Reply:
It works on v1.x but it is broken on `main` / v2.x. I think it broken during a refactor to testnode, potentially https://github.com/celestiaorg/celestia-app/pull/1991/files#diff-1408d16ad448ff1be464a1eb5af128736aca3302ba3a5d7fbebd7de86473a478R107